hadoop-hdds/common/src/main/java/org/apache/hadoop/hdds/client/OzoneQuota.java

```diff
@@ -19,52 +19,82 @@
 package org.apache.hadoop.hdds.client;
 
 import com.google.common.base.Strings;
+import org.apache.hadoop.ozone.OzoneConsts;
+import org.apache.ratis.util.Preconditions;
 
-import static org.apache.hadoop.ozone.OzoneConsts.GB;
-import static org.apache.hadoop.ozone.OzoneConsts.KB;
-import static org.apache.hadoop.ozone.OzoneConsts.MB;
-import static org.apache.hadoop.ozone.OzoneConsts.TB;
-
+import java.util.ArrayList;
+import java.util.Arrays;
+import java.util.Collections;
+import java.util.List;
 
 /**
  * represents an OzoneQuota Object that can be applied to
  * a storage volume.
  */
 public final class OzoneQuota {
 
-  public static final String OZONE_QUOTA_B = "B";
-  public static final String OZONE_QUOTA_KB = "KB";
-  public static final String OZONE_QUOTA_MB = "MB";
-  public static final String OZONE_QUOTA_GB = "GB";
-  public static final String OZONE_QUOTA_TB = "TB";
-
   /** Quota Units.*/
-  public enum Units { B, KB, MB, GB, TB }
+  public enum Units {
+    // the names and the ordering are important
+    B(1),
+    KB(OzoneConsts.KB),
+    MB(OzoneConsts.MB),
+    GB(OzoneConsts.GB),
+    TB(OzoneConsts.TB);
+
```

Comment:
Let's add also 
```java
+    PB(OzoneConsts.PB),
+    EB(OzoneConsts.EB);
// need to add OzoneConsts.PB and OzoneConsts.EB
```
It sounds stupid if someone wants to set 1PB quote but Ozone does not support it.

Also, after the above change, the following will never have `ArrayIndexOutOfBoundsException`.
```java
//RawQuotaInBytes.valueOf(..)
      final int i = Long.numberOfTrailingZeros(quotaInBytes) / 10;
      final Units unit = Units.values()[i];
```